test/integration/ServerTests/AppiumLocalServerLaunchingTest.cs

```diff
@@ -41,25 +37,8 @@ public void BeforeAll()
             }
             Console.WriteLine(_testIp);
 
-            byte[] bytes;
-            if (isWindows)
-            {
-                bytes = Resources.PathToWindowsNode;
-                _pathToCustomizedAppiumJs = Encoding.UTF8.GetString(bytes);
-                return;
-            }
-            if (isMacOs)
-            {
-                bytes = Resources.PathToMacOSNode;
-                _pathToCustomizedAppiumJs = Encoding.UTF8.GetString(bytes);
-                return;
-            }
-            if (isLinux)
-            {
-                bytes = Resources.PathToLinuxNode;
-                _pathToCustomizedAppiumJs = Encoding.UTF8.GetString(bytes);
-                return;
-            }
+            Paths paths = new Paths();
+            _pathToAppiumPackageIndex = paths.PathToAppiumPackageIndex;
```

Comment:
do we need to keep it as a property here if we get always get the path from the corresponding paths property?

Reply:
Either we keep it as a property or I will need to create the Paths obj each time I need this property.

Reply:
fine with me. I was just confused by the intermediate paths variable. Perhaps we could simply do _pathToAppiumPackageIndex = new Paths().PathToAppiumPackageIndex